packages/warehouses/src/warehouseClients/PostgresWarehouseClient.ts

```diff
@@ -314,6 +314,13 @@ export class PostgresClient<
         if (databases.size <= 0 || schemas.size <= 0 || tables.size <= 0) {
             return {};
         }
+
+        const { rows: supportsMatViewsRows } = await this.runQuery(
+            `SELECT current_setting('server_version_num')::integer >= 120000 as supports_matviews`,
+        );
+        const supportsMatviews =
+            supportsMatViewsRows[0]?.supports_matviews ?? '';
```

Comment:
```suggestion
            supportsMatViewsRows[0]?.supports_matviews ?? false;
```

Should this be a boolean ? rather than an empty string